superset-frontend/spec/javascripts/explore/components/FilterBoxItemControl_spec.jsx

```diff
@@ -26,7 +26,9 @@ import FilterBoxItemControl from 'src/explore/components/controls/FilterBoxItemC
 import FormRow from 'src/components/FormRow';
 import datasources from '../../../fixtures/mockDatasource';
 
+const TEST_LABEL = 'some label';
 const defaultProps = {
+  label: TEST_LABEL,
```

Comment:
As `TEST_LABEL` isn't being used anywhere else, I don't see a reason create a `const` for it.
```suggestion
const defaultProps = {
  label:  'some label',
```

Reply:
fixed